----------------------------------------------------------------------
Revision: 8cb2147f6c474c56b6bea37f7fed2ae37ea7e155
Parent:   880df7b506faa1ac84871b84629616a66bc02b84
Author:   yan12125@gmail.com
Date:     西元2019年06月13日 21時15分35秒
Branch:   net.venge.monotone

Changelog: 

Fix building with botan 2.5+

DEFAULT_BUFFERSIZE is renamed to BOTAN_DEFAULT_BUFFER_SIZE in this commit:
https://github.com/randombit/botan/commit/fb44fe4f9ac094a7f223ef337796c3c4a2ed933f

Changes against parent 880df7b506faa1ac84871b84629616a66bc02b84

  patched  src/gzip.cc

============================================================
--- src/gzip.cc	9d7f68b36bf9039a8138317603abe37db5451d66
+++ src/gzip.cc	7872a4ddd5d805731cfaba0a7ff743224763f3fc
@@ -88,7 +88,7 @@ Gzip_Compression::Gzip_Compression(u32bi
 * Gzip_Compression Constructor                   *
 *************************************************/
 Gzip_Compression::Gzip_Compression(u32bit l) :
-   level((l >= 9) ? 9 : l), buffer(DEFAULT_BUFFERSIZE),
+   level((l >= 9) ? 9 : l), buffer(BOTAN_DEFAULT_BUFFER_SIZE),
    pipe(new Hash_Filter("CRC32")), count( 0 )
    {
 
@@ -220,11 +220,11 @@ void Gzip_Compression::put_footer()
 /*************************************************
 * Gzip_Decompression Constructor                 *
 *************************************************/
-Gzip_Decompression::Gzip_Decompression() : buffer(DEFAULT_BUFFERSIZE),
+Gzip_Decompression::Gzip_Decompression() : buffer(BOTAN_DEFAULT_BUFFER_SIZE),
    no_writes(true), pipe(new Hash_Filter("CRC32")), footer(0)
    {
-   if (DEFAULT_BUFFERSIZE < sizeof(GZIP::GZIP_HEADER))
-      throw Decoding_Error("DEFAULT_BUFFERSIZE is too small");
+   if (BOTAN_DEFAULT_BUFFER_SIZE < sizeof(GZIP::GZIP_HEADER))
+      throw Decoding_Error("BOTAN_DEFAULT_BUFFER_SIZE is too small");
 
    zlib = new Zlib_Stream;
 
